src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.css

```diff
@@ -0,0 +1,52 @@
+.tokenAmount {
+  display: grid;
+  width: 100%;
+  grid-template-columns: auto 100px;
+}
+
+.tokenAmountSelect {
+  align-self: center;
+  margin-left: 20px;
+}
+
+.tokenAmountUsd {
+  justify-self: right;
+}
+
+.domainPotBalance {
+  font-size: var(--size-small);
+  font-weight: 600;
+  color: var(--temp-grey-blue-7);
+}
+
+.domainSelects {
+  display: grid;
+  width: 100%;
+  grid-template-columns: calc(50% - 25px) 50px calc(50% - 25px);
+}
+
+.transferIcon {
+  align-self: center;
+  height: 26px;
+  width: 26px;
+  position: relative;
+  top: -7px;
+  justify-self: center;
+
+  & svg {
```

Comment:
While this is technically possible currently via our webpack loaders, future updates might break it.

Please move it to a new class declaration. Something like:
```js
.transferIcon svg {
  fill: var(--temp-grey-blue-7);
}
```